src/Components/lst-Linegraph.js

```diff
@@ -1,49 +1,57 @@
-import React from 'react';
-import {Line} from 'react-chartjs-2';
-const options = {
-  legend: {
-    display: true,
-    position: "bottom",
-  },
-  }
-const state = {
-  labels: ['Q1', 'Q2', 'Q3',
-           'Q4'],
-  datasets: [
-    {
-      label: 'National land surface temperature',
-      fill: false,
-      backgroundColor: 'rgba(75,192,192,1)',
-      borderColor: 'rgba(75,192,192,1)',
-      borderWidth: 2,
-      data: [1162, 289, 1170, 981]
-    },
-    {
-      label: ' District land surface temperature',
-      fill: false,
-      backgroundColor: 'rgba(255,99,132,1)',
-      borderColor: 'rgba(255,99,132,1)',
-      borderWidth: 2,
-      data: [990, 170, 840, 1176]
-    }
-  ]
-}
-
-export default class Temperature extends React.Component {
+import React from "react";
+import { connect } from "react-redux";
+import { Line } from "react-chartjs-2";
+class Temperature extends React.Component {
   render() {
+    const options = {
```

Comment:
Can you help me also understand why this const variables were moved into the class?
Please don't push anything to this branch because I'm gathering the changes into a more concise branch.

Reply:
@lambert3 , you never got back to me on this.